paderbox/utils/pretty.py

```diff
@@ -71,7 +71,7 @@ def pprint(
      - Support multiple objects (Causes bad readable error in original)
 
 
-    >>> pprint([np.array([1]), np.array([1]*100)])
+    >>> pprint([np.array([1]), np.array([1]*100).astype(np.int64)])
```

Comment:
Can you change this to `pprint([np.array([1.]), np.array([1.]*100)])`?
It is not important to use integers here, float64 is also fine.